### PR TITLE
Project management app

### PR DESCRIPTION
**Pull request by sknepali**

### CODE REVIEW

File: Project-14-Project-management-system/.vite/deps/_metadata.json
Comment (SujalShah3234) [inline Project-14-Project-management-system/.vite/deps/_metadata.json:1]: @sknepali Confirm if `.vite` folder is required to push on Repo
If not add it in ignore file and exclude it from commit

File: Project-14-Project-management-system/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
```
Comment (SujalShah3234) [inline Project-14-Project-management-system/package.json:13]: @sknepali Remove `^` from all dependencies and give reason for it.

File: Project-14-Project-management-system/src/assets/no-projects.png
Comment (SujalShah3234) [inline Project-14-Project-management-system/src/assets/no-projects.png:1]: @sknepali Is it the same image as logo?
Remove the duplicate assets

File: Project-14-Project-management-system/src/components/Tasks.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  const currentProjectTasks = tasks.filter(
+    (task) => task.projectId === selectedProjectId
+  );
+
+  return (
+    <section>
+      <h2 className="text-2xl font-bold text-stone-700 mb-4">Tasks</h2>
+      <NewTask onAdd={onAddTask} />
+      {currentProjectTasks.length === 0 && (
```
Comment (SujalShah3234) [inline Project-14-Project-management-system/src/components/Tasks.jsx:20]: @sknepali use `?` operator

File: Project-14-Project-management-system/src/store/projectReducer.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import { CURRENT_ACTION } from "../config";
+
+const [ADDING, SELECTED_PROJECT, NOTHING_SELECTED] = CURRENT_ACTION;
+
+const projectReducer = (state, action) => {
```
Comment (SujalShah3234) [inline Project-14-Project-management-system/src/store/projectReducer.jsx:5]: @sknepali Use `switch case` here

**Review by SujalShah3234 — CHANGES_REQUESTED** [review]

File: Project-14-Project-management-system/package.json (hunk 1 above)
Comment (sknepali) [reply Project-14-Project-management-system/package.json:13]: The caret (^) symbol indicates a "compatible" update. It allows for updates that do not include breaking changes. 
```suggestion
    "react": "^18.2.0",
```

It signifies that the project can use any version greater than or equal to 18.2.0 but less than the next major version.

File: Project-14-Project-management-system/src/components/Tasks.jsx (hunk 2 above)
Comment (SujalShah3234) [reply Project-14-Project-management-system/src/components/Tasks.jsx]: @sknepali ^^

**Pull request merged**